README.md

```diff
@@ -1 +1,104 @@
+## 📗 Table of Contents
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+ - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+ - [👥 Authors](#authors)
+ - [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+# 📖  Mobile First Portfolio <a name="about-project"></a> 
+ 
+Mobile First Portfolio is a mobile website designed as part of Microverse coursework.
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML and CSS
+
+### Tech Stack <a name="tech-stack"></a>
+HTML and CSS
+
+<!-- Features -->
+### Key Features <a name="key-features"></a>
+- The toolbar
+- Headline section
+- Styling
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+clone from repository: https://github.com/Enoisong/Portfolio--Setup-and-Mobile-First
+
+### Prerequisites
+In order to run this project you need to:
+Install Live server VScode extension and run Go Live
+
+### Setup
+Clone this repository to your desired folder: 
+https://github.com/Enoisong/Mobile-First-Portfolio.git
+
+### Install
+Install this project with:
+
+ VSCode
+
+### Usage
+
+To run the project:  
+Install the Live server extension of the VScode and run Go Live
+
+### Run tests
+
+To run linters tests, run the following command: 
+npx hint .
+npx stylelint "**/*.{css,scss}"
+
+<!-- AUTHOR -->
+## 👥 Author <a name="author"></a> 
+Enobong Isong
+
+👤 **Author**
+- GitHub: [@githubhandle](https://github.com/Enoisong)
+- Twitter: [@twitterhandle](https://twitter.com/Enobongmisong)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/enobong-isong/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add List of Portfolio  
+- Add Contact Form  
+- Add Footer  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+If you like this project, give me a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to acknowledge Microverse community for thier support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+ 
+## 📝 License <a name="license"></a> 
+
+This project is [MIT](./MIT.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] [OPTIONAL] It would be nice if you remove this `guide` lines `101` to `103` 👍🏼